geo-types/src/multi_line_string.rs

```diff
@@ -34,6 +34,36 @@ pub struct MultiLineString<T>(pub Vec<LineString<T>>)
 where
     T: CoordinateType;
 
+impl<T: CoordinateType> MultiLineString<T> {
+    /// True if the MultiLineString is non-empty and each of its LineStrings is closed - that is,
+    /// that the first and last coordinates of each line string are the same.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use geo_types::{MultiLineString, LineString, line_string};
+    ///
+    /// let open_line_string: LineString<f32> = line_string![(x: 0., y: 0.), (x: 5., y: 0.)];
+    /// assert!(!MultiLineString(vec![open_line_string.clone()]).is_closed());
+    ///
+    /// let closed_line_string: LineString<f32> = line_string![(x: 0., y: 0.), (x: 5., y: 0.), (x: 0., y: 0.)];
+    /// assert!(MultiLineString(vec![closed_line_string.clone()]).is_closed());
+    ///
+    /// // MultiLineString is not closed if *any* of it's LineStrings are not closed
+    /// assert!(!MultiLineString(vec![open_line_string, closed_line_string]).is_closed());
+    ///
+    /// // An empty MultiLineString is not closed
+    /// assert!(!MultiLineString::<f32>(vec![]).is_closed());
+    /// ```
+    pub fn is_closed(&self) -> bool {
+        if self.0.is_empty() {
```

Comment:
Note this bit @rmanoka - it mirrors [JTS's MultiLineString#is_closed](https://github.com/locationtech/jts/blob/master/modules/core/src/main/java/org/locationtech/jts/geom/MultiLineString.java#L74), but seems to be on similarly shaky ground.

Reply:
@michaelkirk Ya, not a fan of returning `false` if empty here either.  Usually, it is mathematically neat to extend predicates on list that are defined as a for-all over elements in the list to be true on the empty list.  See for instance [std::Iterator::all](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.all).  Since is_closed is such a predicate, I would have liked it to be true on empty list.  I have no idea why JTS chose the other way.  There is no MultiLinearRing in JTS, so I don't have a compelling argument either :( .

Reply:
Ultimately, I can't find a reason why it'd matter, so I'm willing to defer to your intuition here.